frontend/dockerfile/parser/fuzz.go

```diff
@@ -0,0 +1,24 @@
+package parser
+
+import (
+	"os"
+)
+
+func Fuzz(data []byte) int {
+	f, err := os.Create("Dockerfile")
+	if err != nil {
+		return -1
+	}
+	defer f.Close()
+	defer os.Remove("Dockerfile")
+	_, err = f.Write(data)
+	if err != nil {
+		return -1
+	}
+	f.Seek(0, 0)
+	_, err = Parse(f)
```

Comment:
`Parse(io.Reader)` does not need the file to be created. Just  passing`bytes.Buffer`should be enough.